### PR TITLE
splitting Metropolis into Byzantium and Constantinople

### LINKED ISSUE #4268: Split Metropolis into two
In the previous all core dev meeting, I sensed the consensus to split Metropolist into two.  EIP-86 (account abstraction) is moved into the second phase of Metropolis.  This needs to be implemented.

### PR DESCRIPTION
**Pull request by pirapira**
Fixes #4268 .

### CODE REVIEW

File: libethashseal/genesis/test/mainNetworkTest.cpp [hunk 1]
```diff
@@ -26,7 +26,8 @@ R"E(
 		"daoHardforkBlock": "0x1d4c00",
 		"EIP150ForkBlock": "0x259518",
 		"EIP158ForkBlock": "0x28d138",
-		"metropolisForkBlock": "0x2dc6c0",
+		"byzantiumForkBlock": "0x2dc6c0",
+		"constantinopleForkBlock": "0x2dc6c0",
```
Comment (winsvega) [inline libethashseal/genesis/test/mainNetworkTest.cpp]: need a higher value here to split the fork activation point

File: libethashseal/genesis/test/transitionnetTest.cpp [hunk 2]
```diff
@@ -26,7 +26,8 @@ R"E(
 		"daoHardforkBlock": "0x08",
 		"EIP150ForkBlock": "0x0a",
 		"EIP158ForkBlock": "0x0e",
-		"metropolisForkBlock": "0x10",
+		"byzantiumForkBlock": "0x10",
+		"constantinopleForkBlock": "0x10",
```
Comment (winsvega) [inline libethashseal/genesis/test/transitionnetTest.cpp]: same here

**Comment by pirapira** [comment]
@winsvega I delayed Constantinople.

**Comment by pirapira** [comment]
I will continue adjusting evmjit.

@winsvega are you going to fix the tests?

**Comment by winsvega** [comment]
yes

**Comment by pirapira** [comment]
I updated evmjit.

**Pull request merged**